cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/KafkaSampleStore.java

```diff
@@ -234,14 +234,15 @@ private void increaseTopicReplicationFactor(ZkUtils zkUtils,
     Map<Integer, String> rackByBroker = new HashMap<>();
     for (BrokerMetadata bm:
          JavaConversions.seqAsJavaList(AdminUtils.getBrokerMetadatas(zkUtils, RackAwareMode.Enforced$.MODULE$, Option.empty()))) {
-      String rack = bm.rack().get();
+      // If the rack is not specified, we use the broker id info as rack info.
+      String rack = bm.rack().isEmpty() ? String.valueOf(bm.id()) : bm.rack().get();
```

Comment:
But then if a host has multiple brokers, we will violate rack awareness. 
Can we either use (1) metadata request as used in `LoadMonitor` or (2) explicitly specify this limitation in the documentation of this method?